project/Dependencies.scala

```diff
@@ -57,7 +57,7 @@ object Dependencies {
   }
 
   object reactivemongo {
-    val driver = "org.reactivemongo" %% "reactivemongo"                              % "1.1.0-RC11"
```

Comment:
uh I don't think this belongs here

Reply:
oops, my bad. good catch. by the way rc12 and mongo7 are a match made in heaver so far

Reply:
maye the prod gods hear you. I have my doubts

Reply:
we should try it with prod data on an aws setup or something. we can't fake the load but lukhas will thoroughly enjoy the upgrade process 5 -> 6 -> 7 so he should be allowed to repeat it as many times as possible.